From 1c8331829ea6a9d8b992925614624263303f0b29 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:26 -0400
Subject: [ethernet] igb: rename igb_ptp_enable to igb_ptp_feature_enable

Message-id: <1409062359-13063-69-git-send-email-sassmann@redhat.com>
Patchwork-id: 89183
O-Subject: [RHEL7.1 PATCH 68/81] igb: rename igb_ptp_enable to igb_ptp_feature_enable
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jacob Keller <jacob.e.keller@intel.com>

The name igb_ptp_enable is not synonymous with the purpose of this
function, so rename it to better explain its purpose.

Signed-off-by: Jacob Keller <jacob.e.keller@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 102be52ffdb2f7b33c3122409147a05c3e7860ea)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/igb_ptp.c b/drivers/net/ethernet/intel/igb/igb_ptp.c
index ab25e49..e291be2 100644
--- a/drivers/net/ethernet/intel/igb/igb_ptp.c
+++ b/drivers/net/ethernet/intel/igb/igb_ptp.c
@@ -360,8 +360,8 @@ static int igb_ptp_settime_i210(struct ptp_clock_info *ptp,
  return 0;
 }
 
-static int igb_ptp_enable(struct ptp_clock_info *ptp,
-     struct ptp_clock_request *rq, int on)
+static int igb_ptp_feature_enable(struct ptp_clock_info *ptp,
+      struct ptp_clock_request *rq, int on)
 {
  return -EOPNOTSUPP;
 }
@@ -745,7 +745,7 @@ void igb_ptp_init(struct igb_adapter *adapter)
   adapter->ptp_caps.adjtime = igb_ptp_adjtime_82576;
   adapter->ptp_caps.gettime = igb_ptp_gettime_82576;
   adapter->ptp_caps.settime = igb_ptp_settime_82576;
-  adapter->ptp_caps.enable = igb_ptp_enable;
+  adapter->ptp_caps.enable = igb_ptp_feature_enable;
   adapter->cc.read = igb_ptp_read_82576;
   adapter->cc.mask = CLOCKSOURCE_MASK(64);
   adapter->cc.mult = 1;
@@ -765,7 +765,7 @@ void igb_ptp_init(struct igb_adapter *adapter)
   adapter->ptp_caps.adjtime = igb_ptp_adjtime_82576;
   adapter->ptp_caps.gettime = igb_ptp_gettime_82576;
   adapter->ptp_caps.settime = igb_ptp_settime_82576;
-  adapter->ptp_caps.enable = igb_ptp_enable;
+  adapter->ptp_caps.enable = igb_ptp_feature_enable;
   adapter->cc.read = igb_ptp_read_82580;
   adapter->cc.mask = CLOCKSOURCE_MASK(IGB_NBITS_82580);
   adapter->cc.mult = 1;
@@ -784,7 +784,7 @@ void igb_ptp_init(struct igb_adapter *adapter)
   adapter->ptp_caps.adjtime = igb_ptp_adjtime_i210;
   adapter->ptp_caps.gettime = igb_ptp_gettime_i210;
   adapter->ptp_caps.settime = igb_ptp_settime_i210;
-  adapter->ptp_caps.enable = igb_ptp_enable;
+  adapter->ptp_caps.enable = igb_ptp_feature_enable;
   /* Enable the timer functions by clearing bit 31. */
   wr32(E1000_TSAUXC, 0x0);
   break;
-- 
1.7.1